app/views/static_pages/help.html.erb

```diff
@@ -18,6 +18,13 @@
             <%= sanitize(_('If you need assistance in developing your data management plan, the following resource may help get you started: <a href="%{how_to_manage_your_data_url}"> How to manage your data</a>. If you require more guidance, contact us at <a href="mailto:%{email_link}?Subject=%{email_subject}">dmp-assistant@tech.alliancecan.ca</a>.')  % {contact_your_institution_url: contact_your_institution_url, email_link: email_link, email_subject: email_subject, how_to_manage_your_data_url: how_to_manage_your_data_url}) %>
           </p>
 
+          <p>
+            <% sso_english_guidlines_url = sso_english_guidlines_path%>
+            <% sso_french_guidlines_url = sso_french_guidlines_path%>
+            <%= sanitize(_('For help with single sign-on, help documents are available in <a href="%{sso_english_guidlines_url}"> English </a> and in <a href="%{sso_french_guidlines_url}"> French</a>.') %
+            {sso_french_guidlines_url: sso_french_guidlines_url, sso_english_guidlines_url: sso_english_guidlines_url } )%>
```

Comment:
I like Omar's refactoring idea https://github.com/portagenetwork/roadmap/pull/878/files#r1744051419. If you make that change, we should also just fix the `_guidlines_` typo to `_guidelines_`.